blocks/identity-block/features/account-management/_children/SocialSignOnEditableContainer.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect } from "react";
+
+import { Button, usePhrases, Icon, useIdentity } from "@wpmedia/arc-themes-components";
+
+function SocialSignOnEditableFieldContainer({
+	blockClassName,
+	socialIcon,
+	type,
+	isConnected,
+	onDisconnectFunction,
+}) {
+	const phrases = usePhrases();
+	const { Identity } = useIdentity();
+
+	const googleNotification = phrases.t("identity-block.social-google-one-tap-notification");
+
+	const connectedText = phrases.t("identity-block.connected-identity");
+	const connectText = phrases.t("identity-block.connect-identity");
+	const disconnectText = phrases.t("identity-block.disconnect-account");
+
+	useEffect(() => {
+		if (type === "Google" && !isConnected) {
+			const attachEventListener = () =>
+				document.getElementById("custom-Google-signin-btn").addEventListener("click", () =>
+					window.google.accounts.id.prompt((notification) => {
+						if (notification.isNotDisplayed() || notification.isSkippedMoment()) {
+							// https://developers.google.com/identity/gsi/web/reference/js-reference
+							// https://developers.google.com/identity/gsi/web/guides/features#exponential_cooldown
+							// eslint-disable-next-line
+							alert(googleNotification);
+							// Remove cookie works in Safari
+							document.cookie =  `g_state=;path=/;expires=Thu, 01 Jan 1970 00:00:01 GMT`;
+							window.google.accounts.id.prompt();
+						}
+					}),
+				);
```

Comment:
Would rather see this as an onClick in the button below with a switch for the "Google" type.  Then you would not need the 0 length timer to push this to the end of the call stack as well... see below.